packages/manager/apps/support/index.routing.js

```diff
@@ -0,0 +1,18 @@
+import template from './root.html';
+
+export const APP_STATE = {
+  abstract: true,
+  name: 'app',
+  redirectTo: 'app.account',
+};
+
+export const APP_ACCOUNT_STATE = {
+  name: 'app.account',
+  redirectTo: 'app.account.support',
+  template,
+};
```

Comment:
Imo, we should try avoid multiple state declaration in a file

Reply:
We plan to address this in the second wave. Thanks for the inputs!